platform/viewer/src/components/Header/Header.js

```diff
@@ -36,18 +36,14 @@ class Header extends Component {
   }
 
   loadOptions() {
-    const {
-      t,
-      user,
-      userManager,
-      modal: { show },
-    } = this.props;
+    const { t, user, userManager, modal } = this.props;
```

Comment:
Can we use `withTranslations`?

Reply:
Which one? currently its using withTranslation from react-i18next

Reply:
Pointing out that we may not need to pass `t` as a prop